learn/building_programs/index.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: book
+title: Building programs
+permalink: /learn/building_programs
+date: 2020-06-05
+author: Arjen Markus

```

Comment:
Add co-authors to the list, sorted alphabetically by last name. Did I miss anybody? Please add your name if so.

```suggestion
author: Arjen Markus, Ondřej Čertík, Milan Curcic, Laurence Kedward, Brad Richardson, Jeremie Vandenplas
```